index.ts

```diff
@@ -0,0 +1,33 @@
+import * as Botkit from 'botkit';
+import axios from 'axios';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const bot = Botkit.slackbot({});
+bot.spawn({
+  token: process.env.SLACK_TOKEN,
+}).startRTM((err) => {
+  if (err) {
+    throw err;
+  }
+});
+
+bot.hears(/now/, ['mention', 'direct_mention'], (bot, msg) => {
+  axios.request({
+    url: process.env.KURUO_URL,
+    method: 'get',
+    responseType: 'stream'
+  }).then((res) => {
+    const payload = {
+      channels: msg.channel,
+      filename: 'kuruo.jpg',
+      title: 'kuruo_now',
+      file: res.data,
+    };
+
+    bot.api.files.upload(payload, (err, _) => {
+      console.log(err);
```

Comment:
:nits: This is good enough but should post error log to Slack channel?

example:
```js
bot.dialogError({
  "name": "kuruo_error",
  "error": err
});
```

_I don't understand Botkit..._

Reply:
https://botkit.ai/docs/readme-slack.html#botdialogerror